tox.ini

```
@@ -2,16 +2,17 @@
 min_version =  4.11
 
 env_list =
-    python{3.8,3.9,3.10,3.11}-django4.2-wagtail{5.2,6.0,6.1}
-    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.0,6.1}
+    python{3.9,3.10,3.11}-django4.2-wagtail{5.2,6.2,6.3}
+    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.2,6.3}
+    python{3.10,3.11,3.12,3.13}-django5.1-wagtail6.3
```

```suggestion
    python{3.12,3.13}-django5.1-wagtail6.3
```

will save some compute cycles